DESCRIPTION

```diff
@@ -23,8 +23,16 @@ Authors@R:
            email = "sebastian.funk@lshtm.ac.uk",
            comment = c(ORCID = "0000-0002-2842-3406")))
 Description: Provides functions for nowcasting right-truncated epidemiological
```

Comment:
I think we need baseline or simple or something drawing on the justification given in the abstract of the write up very early on. 

I don't think we need to link out to the KIT model here (and links in general are annoying for descriptions when on CRAN). If we are referencing them it should be later once we have established why people care. I think early on we need to link to epinowcast processing as that is a core aim.

Think reason, functionality (from high to low level), how.

Reply:
I removed the reference to the KIT model. 

Confused by what you mean about linking to epinowcast processing, can you add a suggestion? I have revised the description.